modules/migration-compatibility-guidelines-3-to-4.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/installing-3-4.adoc
+// * migrating_from_ocp_3_to_4/installing-restricted-3-4.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="migration-compatibility-guidelines-3-to-4_{context}"]
+= Compatibility guidelines
+
+You must install the {mtc-full} ({mtc-short}) Operator that is compatible with your {product-title} version.
+
+.Definitions
+
+control cluster:: The cluster that runs the {mtc-short} controller and GUI.
+remote cluster:: A source or destination cluster for a migration that runs Velero. The Control Cluster communicates with Remote clusters using the Velero API to drive migrations.
+
+You must use the compatible {mtc-short} version for migrating your {product-title} clusters. For the migration to succeed both your source cluster and the destination cluster must use the same version of {mtc-short}.
+
+MTC 1.7 supports migrations from {product-title} 3.11 to 4.9.
```

Comment:
Sorry if this wasn't clear, but 1.7 still expects you to be on a modern 4.x version. In fact I believe our current support policy expects the user to be installing the latest OCP 4 version

Reply:
@dymurray So MTC 1.7 supports migrating from OCP 3.11 to any version of OCP 4, right? The sentence about MTC 1.8 is there because users need to install MYC 1,8 on the destination 4.x cluster, right?